src/DynamoCore/DynamoCore.csproj

```diff
@@ -293,6 +294,7 @@ limitations under the License.
       <LastGenOutput>Resources.en-US.Designer.cs</LastGenOutput>
       <SubType>Designer</SubType>
     </EmbeddedResource>
+    <EmbeddedResource Include="ProtoGeometryImages.resx" />
```

Comment:
One more thing, please make sure the `ProtoGeometry.customization.dll` is in the right folder (not inside the `nodes` folder because that is where `GeometryUiWpf.dll` is generated).